src/components/Parameters/Parameters.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import BodySchema from '../BodySchema/BodySchema'
+
+export default class Parameters extends PureComponent {
+  render () {
+    const {parameters, classes} = this.props
+
+    if (!parameters) {
+      return null
+    }
+
+    const parameterUIElements = []
+    for (const key in parameters) {
```

Comment:
Curious about this approach of creating the array here and then using it later. Other components generally do this inline with `.map`. Was there any particular reason this was avoided in this case? (eg. `Object.keys(parameters).map(...)`)